sympy/algebras/tests/test_quaternion.py

```diff
@@ -50,11 +51,15 @@ def test_quaternion_complex_real_addition():
 def test_quaternion_functions():
     q = Quaternion(x, y, z, w)
     q1 = Quaternion(1, 2, 3, 4)
+    q0 = Quaternion(0, 0, 0, 0)
 
     assert conjugate(q) == Quaternion(x, -y, -z, -w)
     assert q.norm() == sqrt(w**2 + x**2 + y**2 + z**2)
     assert q.normalize() == Quaternion(x, y, z, w) / sqrt(w**2 + x**2 + y**2 + z**2)
     assert q.inverse() == Quaternion(x, -y, -z, -w) / (w**2 + x**2 + y**2 + z**2)
+    def test1(p):
+        p1 = p.inverse()
+    raises(ValueError, lambda: test1(q0))
```

Comment:
There should be no need to write a `test1` function to test this change, it can be tested without it.